README.md

```diff
@@ -1,2 +1,115 @@
-# transbank-sdk-ios-onepay
-SDK para integración de OnePay en app nativas Android
+# Transbank SDK Android Onepay
+
+## Instalación
+
+- Descarga el SDK desde desde [la página de releases](https://github.com/TransbankDevelopers/transbank-sdk-ios-onepay/releases). Para comenzar el desarrollo puedes usar la versión para ambiente de Test (y luego deberás compilar contra la versión de Producción para tu app productiva)
```

Comment:
@leosoto Maybe we should specify which file must be downloaded from the Releases page. I'm saying this because there are more files to download, like "Source code.zip", which only contains this README.md.

Reply:
Unfortunately Github generates the "source code" itself and there is no way to avoid it. So I put in the release pages the text:

> Release solo binario del SDK provisto por IONIX. Son binarios diferentes por cada ambiente:

> - OnePaySDK - QA-TBK.framework.zip: Ambiente de Testing
> - OnePaySDK - PROD-TBK.framework.zip: Ambiente de Producción

But sure it won't hurt if I'm more specific on the README too